src/main/java/org/apache/sling/auth/oauth_client/ClientConnection.java

```diff
@@ -34,4 +35,10 @@ public interface ClientConnection {
      * @return the name of the connection
      */
     @NotNull String name();
+    String clientId();
```

Comment:
@nscendoni - the `ClientConnection` was designed to not leak implementation details to the client. What was the reason to add all these field to it?

BTW, it does not seem related to the 'move to Java 11' commit :-)